lib/utils/index.js

```diff
@@ -200,34 +182,44 @@ module.exports = {
   isCustomComponent (node) {
     assert(node && node.type === 'VElement')
 
-    const name = node.name
     return (
-            !(this.isHtmlElementName(name) || this.isSvgElementName(name)) ||
-            this.hasAttribute(node, 'is') ||
-            this.hasDirective(node, 'bind', 'is')
+      !(this.isHtmlElementNode(node) || this.isSvgElementNode(node) || this.isMathMLElementNode(node)) ||
+      this.hasAttribute(node, 'is') ||
+      this.hasDirective(node, 'bind', 'is')
     )
   },
 
   /**
-   * Check whether the given name is a HTML element name or not.
-   * @param {string} name The name to check.
-   * @returns {boolean} `true` if the name is a HTML element name.
+   * Check whether the given node is a HTML element or not.
+   * @param {ASTNode} node The node to check.
+   * @returns {boolean} `true` if the node is a HTML element.
    */
-  isHtmlElementName (name) {
-    assert(typeof name === 'string')
+  isHtmlElementNode (node) {
```

Comment:
Thank you for good change.

However, it might cause confusing if `isHtmlElementNode` is not checking their namespace but `isSvgElementNode` and `isMathMLElementNode` are checking their namespace. How about renaming to `isKnownHtmlElementNode`?